Sources/StreamChat/StateLayer/Chat.swift

```diff
@@ -675,57 +700,67 @@ public class Chat {
 
     // MARK: -
     
-    /// Loads replies for the first page of the specified message and updates ``MessageState/replies``.
+    /// Loads more replies before the specified reply id and updates ``MessageState/replies``.
     ///
-    /// - Note: Loading the first page resets the ``MessageState/replies``.
+    /// - Parameters:
+    ///   - replyId: The message id of the reply from which older messages are loaded. If nil, the oldest currently loaded message id in ``MessageState/replies`` is used.
+    ///   - parentMessageId: The parent message id which has replies.
+    ///   - limit: The limit for the page size. The default limit is 25.
+    ///
+    /// - Throws: An error while communicating with the Stream API.
+    public func loadReplies(before replyId: MessageId?, of parentMessageId: MessageId, limit: Int? = nil) async throws {
+        let messageState = try await makeMessageState(for: parentMessageId)
+        return try await messageUpdater.loadReplies(of: parentMessageId, before: replyId, limit: limit, cid: cid, paginationStateHandler: messageState.replyPaginationHandler)
+    }
```

Comment:
Also considered 
`public func loadReplies(of parentMessageId: MessageId, before replyId: MessageId?, limit: Int? = nil) async throws` but it did not read so well.